.github/bump-version.sh

```diff
@@ -30,3 +30,6 @@ echo "$GLOBAL_JSON" > global.json
 
 sed -i'' -E "s/^([[:space:]]+VERSION: )([0-9]+\.){2}[0-9]+$/\1$VERSION/" .github/workflows/release.yml
 sed -i'' -E "s/<(Current(Assembly(File)?)?Version)>([0-9]+\.){2}[0-9]+<\/\1>/<\1>$VERSION<\/\1>/" Directory.Build.props
+
+# Update packages.lock.json
```

Comment:
Do you want to get rid of lock files instead?

Reply:
Have made a PR #265